Leave/leave_interface.py

```diff
@@ -57,20 +56,24 @@ async def HandleLeaveReactions(client, payload):
     if status == None:
         return
 
-    if utils.isNotBot(payload.member) and utils.IsAdmin(payload.member) and leave_db.IsLeaveRequestPending(payload.message_id):
+    if not (utils.isNotBot(payload.member) and utils.IsAdmin(payload.member)):
+        return
+        
+    if status == "Reverted":
+        if not (leave_db.IsLeaveRequestPending(payload.message_id)):
+            leave_db.UpdateLeaveStatus(payload.message_id, "Pending")
+            await UI.UpdateLeaveEmbed(payload.member, message, embed, "Pending", True)
```

Comment:
probably want to inform the applicant about the leave status here

Reply:
Done